src/generateAppendClassName.js

```diff
@@ -1,4 +1,4 @@
-import Map from 'es6-map';
+import Map from './simple-map';
```

Comment:
`Map`, in regular ES2015 maps keys can be objects.

A WeakMap is a map that does not interfere with the garbage collection of its keys. If you'd like to see an example of why one would want to use it - see http://stackoverflow.com/questions/29413222/what-are-the-actual-uses-of-es6-weakmap . The "weakness" of the weak map is in that it holds the reference to the keys weakly.